### PR TITLE
show spiked items in personal view sent items tab

### PR DESCRIPTION
**Pull request by petrjasek**
SDANSA-428

### CODE REVIEW

File: scripts/apps/monitoring/services/CardsService.ts [hunk 1]
```diff
@@ -65,6 +65,8 @@ export function CardsService(search, session, desks, $location) {
 
         if (card.type === 'spike' || card.type === 'spike-personal') {
             params.spike = 'only';
+        } else if (card.type === 'personal' && card.sent) {
+            params.spike = 'include';
```
Comment (tomaskikutis) [inline scripts/apps/monitoring/services/CardsService.ts:69]: I'd call it `include-sent` to be more precise

Comment (petrjasek) [reply scripts/apps/monitoring/services/CardsService.ts:69]: this param just controls if spiked items will be included or excluded, it doesn't handle sent filtering

Comment (tomaskikutis) [reply scripts/apps/monitoring/services/CardsService.ts:69]: it's good to have clear names even for local variables. In this case it's not clear what will be included when that parameter is set.

Comment (petrjasek) [reply scripts/apps/monitoring/services/CardsService.ts:69]: not sure if I follow here, if you set `spike` to `include` it includes spiked items. if it's `spike = 'include-spiked';` that's just duplication, isn't it?

Comment (tomaskikutis) [reply scripts/apps/monitoring/services/CardsService.ts:69]: You're right. It'd be helpful though to document possible values to `spike` option and their meanings.

File: scripts/apps/search/services/SearchService.ts [hunk 2]
```diff
@@ -11,6 +11,7 @@ import {getDateFilters, getDateRangesByKey} from '../directives/DateFilters';
 import {gettext} from 'core/utils';
 import {KILLED_STATES} from 'apps/archive/constants';
 import {appConfig} from 'appConfig';
+import {string} from 'prop-types';
```
Comment (tomaskikutis) [inline scripts/apps/search/services/SearchService.ts:14]: This is probably a mistake, you don't need this import.

**Pull request merged**